### DISCUSSION TITLE
Future work still planned for perseus?

### DISCUSSION THREAD

**Discussion by aethertap** [framesurge/perseus#322] [General]
Hi, I started working on a simple app with perseus and have really enjoyed it so far, but I noticed there hasn't been a code commit for a while. Has development moved to another location, or is there future work still being planned for it here?

**Comment by arctic-hen7** [comment]
Glad to hear you're enjoying it! Honestly life has jut been getting in the way for a while now, and Perseus is no longer my main priority, but there is still development here and there. I would describe the project as passively maintained.

**Reply by aethertap** [reply]
Understood. I really like the way perseus works though, so I'll keep going on this app for a while and see if I can learn enough about the internals to make some useful contributions.

**Reply by arctic-hen7** [reply]
Thanks very much, that would be greatly appreciated! Feel free to post on our Discord if you want any help with the internals!
